lib/src/widgets/chat/ds_file_message_bubble.widget.dart

```diff
@@ -22,6 +22,7 @@ class DSFileMessageBubble extends StatelessWidget {
   final List<DSBorderRadius> borderRadius;
   final DSMessageBubbleStyle style;
   final bool shouldAuthenticate;
+  final String? replyId;
```

Comment:
![image](https://github.com/takenet/blip-ds-flutter/assets/14128784/f77b683f-22b0-4302-8819-8f7d2d40dc9d)

![image](https://github.com/takenet/blip-ds-flutter/assets/14128784/51e26160-0c3b-440f-b338-0110817f1458)

The reply in a doc is not correct in desk app